.travis.yml

```diff
@@ -23,41 +22,53 @@ env:
     - PIP_DEPENDENCIES="codecov"
     - EVENT_TYPE='push pull_request'
     - DEBUG=True
+    - CONDA_CHANNELS="astropy"
```

Comment:
what do we install from conda that we need from the astropy channel?

Reply:
e.g. could just add this to the one build that conda installs `sphinx-astropy` rather than being a global setting (probably doesn't matter much, but I expect things might be slightly be faster using just the default channels)

Reply:
Done